src/lsif.rs

```diff
@@ -260,6 +260,7 @@ pub enum ItemKind {
 #[derive(Debug, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Item {
+    #[serde(rename = "shard")]
     pub document: Id,
```

Comment:
We tend to have to do semver incompatible (major) releases regardless simply due to rust being stricter than what the LSP thinks is a semver compatible change. So just update the field itself to shard instead and I can do a breaking release.

Reply:
Will do!